### DISCUSSION TITLE
Basic question: How do you apply a theme?

### DISCUSSION THREAD

**Discussion by mikwee** [rktjmp/lush.nvim#141] [Q&A]
I know, that's an embarrassingly basic question, but I can't figure it out. I wrote a theme in Lua, now how do I make it the default in Neovim?
Thanks.

**Comment by rktjmp** [comment]
Are you able to run `:colorscheme my_theme` and have it work? If so you just need to put `vim.cmd.colorscheme("my_theme")` into your init.lua, otherwise you need to get the colorscheme into neovims runtimepath some how.

If you just have one colorscheme file, you can probably put it in `~/.config/nvim/colors/my_theme.lua` and have it work, otherwise if you followed something like https://github.com/rktjmp/lush.nvim/blob/main/CREATE.md you probably have a "plugin style" colorscheme (ready for sharing "as a plugin"). You can use a plugin manager to install plugin-style themes, most can install from a local dir as well as a remote git repo.
